gitbook/geostyler/stepSix.md

```diff
@@ -26,6 +26,9 @@ werden. Das `defaultOlStyle` kann somit geparst werden.
 Der `GeoStyler` nimmt nun einen Stil an, jedoch verändert sich dieser beim Scrollen noch nicht. Diese 
 Verknüpfung mit den Daten erfolgt erst im kommenden Kapitel.
 
+***Aufgabe 1.***
+???
```

Comment:
Please remove, if there is no clear task

Reply:
Task removed!